app/uk/gov/hmrc/nisp/connectors/PertaxAuthConnector.scala

```diff
@@ -42,7 +42,7 @@ class PertaxAuthConnectorImpl @Inject()(http: HttpClient, appConfig: Application
   }
 
   override def loadPartial(partialContextUrl: String)(implicit hc: HeaderCarrier): Future[HtmlPartial] = {
-    val partialUrl = appConfig.pertaxAuthBaseUrl + s"/pertax/partials/$partialContextUrl"
+    val partialUrl = appConfig.pertaxAuthBaseUrl + s"/$partialContextUrl"
```

Comment:
partialContextUrl is already prefixed with `/`

Reply:
You were right. I updated the URL string.